### PR TITLE
added delete entry functionality

### PR DESCRIPTION
**Pull request by al-serebrov**

### CODE REVIEW

File: flaskr.py [hunk 1]
```diff
@@ -80,4 +80,16 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
+	return redirect(url_for('show_entries'))
```
Comment (serebrov) [inline flaskr.py]: General note: for some reason the code is over-indented, the base python style guide, [pep8](https://www.python.org/dev/peps/pep-0008/) requires 4 spaces.

File: flaskr.py [hunk 2]
```diff
@@ -80,4 +80,16 @@ def login():
...
+
+@app.route('/delete=<int:entry_id>', methods=['GET','POST'])
+def delete_entry(entry_id):
+	error=None
+	if request.method == 'POST':
```
Comment (serebrov) [inline flaskr.py]: If you define this route with `methods=['POST']`, this check can be removed. Flask will do this and make sure this route is only called with POST method.

File: flaskr.py [hunk 3]
```diff
@@ -80,4 +80,16 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
+	return redirect(url_for('show_entries'))
+
+@app.route('/delete=<int:entry_id>', methods=['GET','POST'])
+def delete_entry(entry_id):
+	error=None
```
Comment (serebrov) [inline flaskr.py]: pep8: `error = None` (spaces around =)

File: flaskr.py [hunk 4]
```diff
@@ -80,4 +80,16 @@ def login():
...
+	if request.method == 'POST':
+		db=get_db()
+		print(entry_id)
+		db.execute('delete from entries where id= (?)', (entry_id,))
```
Comment (serebrov) [inline flaskr.py]: General note: for learning purpose it is OK (and even good) to use DB this way (just execute SQL queries).
But in real application we would use something like SQLAlchemy to perform DB queries.
And for raw SQL I can recommend also this styleguide - http://www.sqlstyle.guide/

**Review by serebrov — APPROVED** [review]
LGTM, nice change, there are few notes, but in general it is good.

File: flaskr.py (hunk 1 above)
Comment (al-serebrov) [reply flaskr.py]: Corrected all pep8 violations using pylint module

File: flaskr.py (hunk 4 above)
Comment (al-serebrov) [reply flaskr.py]: > I can recommend also this styleguide - http://www.sqlstyle.guide/

Thanks, I'll check it out

File: flaskr.py (hunk 2 above)
Comment (al-serebrov) [reply flaskr.py]: Corrected this, thanks

**Pull request merged**